core/shared/src/main/scala/matryoshka/instances/fixedpoint/package.scala

```diff
@@ -201,7 +200,7 @@ package object fixedpoint {
 
     def fromOption[A](opt: Option[A]): Partial[A] = opt.fold(never[A])(now)
 
-    def fromPartialFunction[A, B](pf: scala.PartialFunction[A, B]):
+    def fromPartialFunction[A, B](pf: PartialFunction[A, B]):
```

Comment:
Hrmm, wondering if we want `PartialFunction` in our Predef … (not that it’d be changed in this PR)